response_operations_ui/__init__.py

```diff
@@ -2,11 +2,16 @@
 
 from flask import Flask
 from flask_assets import Bundle, Environment
+from flask_login import LoginManager
 
 from response_operations_ui.logger_config import logger_initial_config
+from response_operations_ui.user import User
 
 
 app = Flask(__name__)
+login_manager = LoginManager(app)
```

Comment:
All this looks good, potentially we could pull all this initialisation around login into it's own file/function? Or at least group it together in this file so it's clear that it's all related

Reply:
I grouped it all together as it's fairly small.  If this ends up being much bigger then I'll split it out.